test/test.js

```diff
@@ -196,3 +196,16 @@ test('Testing the case when an example is provided, use the provided example val
   t.false(/SOME_INTEGER_VALUE/.test(snippet));
   t.end();
 });
+
+test('Query Params Defined for all methods should be resolved', function (t) {
```

Comment:
Test that both params are in method and path. Verify that method overrides path. 
Test get with zero params in method or path.